src/assets/database.sql

```diff
@@ -78,6 +78,35 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
+('File to save subdomains','Write all valid subdomains to the chosen file.', 'dnsenum --subfile subdomains_obtained.txt example.com', 1, 'input', '--subfile', NULL, NULL, 'GENERAL OPTIONS')
+('Timeout setting','The tcp and udp timeout values in seconds. Defines he interval of time in which dnsenum waits from an answer from the DNS server (default: 10s).', 'dnsenum -t 10 example.com', 1, 'input', '-t', NULL, NULL, 'GENERAL OPTIONS')
+('Threads setting', 'The number of threads that will perform different queries.', 'dnsenum --threads 5 example.com', 1, 'checkbox', NULL, NULL, 'GENERAL OPTIONS')
+('Verbose output', 'Be verbose: show all the progress and all the error messages.', NULL, 1, 'checkbox', '-v', NULL, NULL, 'GENERAL OPTIONS')
+--Google Scapping options
+('Google pages scrapping', 'The number of google search pages to process when scraping names, the default is 5 pages, the -s switch must be specified.', 'dnsenum -p 15 - 20 example.com', 1, 'input', '-p', NULL, NULL, 'GOOGLE SCRAPPING OPTIONS')
+('Subdomains scrapping', 'The maximum number of subdomains that will be scraped from Google (default 15)', 'dnsenum -s 40 example.com', 1, 'input', '-s', NULL, NULL, 'GOOGLE SCRAPPING OPTIONS: SUBDOMAINS')
+--Brute force options
+('Set brute force file', 'Read subdomains from this file to perform brute force. (Takes priority over default dns.txt)', 'dnsenum -f subdomains.txt example.com', 1, 'input', '-f', NULL, NULL, 'SUBDOMAIN BRUTE FORCE')
+('Update brute force file', 'This command updates the file specified with the -f switch with the follwing parameters: a (all): Update using all results; g; Update using only google scraping results; r: Update using only reverse lookup results; z: Update using only zonetransfer results.', 'dnsenum -f subdmains.txt -u a example.com', 1, 'input', '-u', NULL, NULL, 'BRUTE FORCE OPTIONS: UPDATE')
+('Recursion on subdomains', 'Executes a brute force in all discovered subdomains that have an NS record.', NULL, 1, 'checkbox', '-r', NULL, NULL, 'BRUTE FORCE OPTIONS: RECURSION')
+--Whois Netrange Options
+('whois queries', 'Perform the whois queries on c class network ranges.**Warning**: this can generate very large netranges and it will take lot of time to perform reverse lookups.', 'dnsenum -w example.com', 1, 'checkbox', NULL, NULL, 'WHOIS NETRANGE OPTIONS')
+('whois queries delay', 'The maximum value of seconds to wait between whois queries, the value is defined randomly, default: 3s.', 'dnsenum -w -d 5 example.com', 1, 'input', '-d', NULL, NULL, 'WHOIS NETRANGE OPTIONS: DELAY')
+--Reverse lookup options
+('PTR records exclusion', 'Exclude PTR records that match the regexp expression from reverse lookup results, useful on invalid hostnames. In this sense, when a reverse lookup is made, subdomains with the defined words are not considered.', 'dnsenum -e '(test|local|invalid)' examples.', 1, 'input', '-e', NULL, NULL, 'REVERSE LOOKUP OPTIONS')
```

Comment:
@mayara-santos01 this will be problematic when executed because of the extra single quotes on attribute 3 (examples). Suggestion:
```suggestion
('PTR records exclusion', 'Exclude PTR records that match the regexp expression from reverse lookup results, useful on invalid hostnames. In this sense, when a reverse lookup is made, subdomains with the defined words are not considered.', 'dnsenum -e "(test|local|invalid)" examples.', 1, 'input', '-e', NULL, NULL, 'REVERSE LOOKUP OPTIONS')
```